app/src/main/res/layout/activity_login.xml

```diff
@@ -1,13 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingStart="@dimen/activity_horizontal_padding"
+    android:paddingBottom="@dimen/activity_vertical_padding"
     android:paddingEnd="@dimen/activity_vertical_padding"
-    android:paddingTop="@dimen/activity_horizontal_padding"
-    android:paddingBottom="@dimen/activity_vertical_padding">
+    android:paddingStart="@dimen/activity_horizontal_padding"
+    android:paddingTop="@dimen/activity_horizontal_padding">
```

Comment:
check the places of the changes you are submitting, 

remember what i told you about the merge "to let all the things from others in his place, and to add yours at the end"